.github/workflows/eslint-analysis.yml

```diff
@@ -0,0 +1,40 @@
+name: ESLint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install ESLint
+        run: |
+          npm i eslint
+          npm i @microsoft/eslint-formatter-sarif
+          
+      - name: Run ESLint
+        run: npx eslint .
+             --config .eslintrc.js
```

Comment:
New to npm/eslint, so i thought it was available since the first tries didn't work.
Removed and it is still working, so we are good :)
Thanks for the review!